README.md

```diff
@@ -46,7 +46,7 @@ go get github.com/SUSE/skuba/cmd/skuba
 
 A development build will:
 
-* Pull container images from `registry.suse.de/devel/caasp/4.0/containers/containers/caasp/v4`
+* Pull container images from `registry.suse.de/devel/caasp/5/containers/cr/containers/caasp/v5/`
```

Comment:
Why uses images in `cr`?  From #1212, I think we should use `registry.suse.de/devel/caasp/5/containers/containers/caasp/v5` for development.

Reply:
Sounds good. Note that this changed after this PR was proposed.

Reply:
resolved